apps/07tic-tac-toe/script.js

```diff
@@ -2,4 +2,170 @@
 
 $(document).on('ready', function() {
     // Put app logic in here
+    
+    var board = [
+    [' ', ' ', ' '],
+    [' ', ' ', ' '],
+    [' ', ' ', ' ']
+];
+    
+    
+    function horizontalWin() {
+    if (board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn ||
+        board[1][0] === playerTurn && board[1][1] === playerTurn && board[1][2] === playerTurn ||
+        board[2][0] === playerTurn && board[2][1] === playerTurn && board[2][2] === playerTurn) {
+        return true;
+    }
+        else {
+            return false;
+        }
+    }
+
+function verticalWin() {
+    if (board[0][0] === playerTurn && board[1][0] === playerTurn && board[2][0] === playerTurn ||
+        board[0][1] === playerTurn && board[1][1] === playerTurn && board[2][1] === playerTurn ||
+        board[0][2] === playerTurn && board[1][2] === playerTurn && board[2][2] === playerTurn) {
+        return true;
+    }
+        else {
+            return false;
+        }
+    }
+
+function diagonalWin() {
+    if (board[0][0] === playerTurn && board[1][1] === playerTurn && board[2][2] === playerTurn ||
+        board[0][2] === playerTurn && board[1][1] === playerTurn && board[2][0] === playerTurn) {
+        return true;
+    }
+        else {
+            return false;
+        }
+    }
+
+    
+    function checkForWin() {
+    if (horizontalWin() || verticalWin() || diagonalWin()) {
+        return true
+        console.log('Player' + playerTurn + 'Won!');
+    } else {
+        return false
+    }
+}
+    
+    function isDraw() {
+    if (board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn &&
+        board[1][0] === playerTurn && board[1][1] === playerTurn && board[1][2] === playerTurn &&
+        board[2][0] === playerTurn && board[2][1] === playerTurn && board[2][2] === playerTurn) {
+        return true
+        console.log('Its a Draw');
+    } else {
+        return false
+    }
+}
+    
+    function checkForDraw() {
```

Comment:
You have two functions called isDraw and checkForDraw. From a developer perspective this is a little confusing as I would expect them both to do the same thing. If one is not needed, simply remove it, otherwise name it in such a way to convey that they are doing different things.